fdbserver/WorkerInterface.actor.h

```diff
@@ -178,7 +180,8 @@ struct RegisterMasterRequest {
 	UID id;
 	LocalityData mi;
 	LogSystemConfig logSystemConfig;
-	std::vector<MasterProxyInterface> proxies;
+	std::vector<MasterProxyInterface> masterProxies;
```

Comment:
I see in other places you removed "master" from names. So it's better to be consistent here, i.e., using "proxies" or "commitProxies".

Reply:
If the name is proxy, then it means GRV proxy and Master proxy altogether. Master proxy now means the proxy sub-role responsible for commit paths.

Reply:
Going forward, we will rename "master" to something else, so it's better not to use "masterProxy" as the name. That's why I suggested using "commitProxies", to indicate the proxy is intended for commits.

Reply:
Sure. We can do the renaming in a separate CL because it's fairly large.

Reply:
Have resolved other comments. Thanks!